mgradm/cmd/migrate/shared/flags.go

```diff
@@ -11,8 +11,10 @@ import (
 
 type MigrateFlags struct {
 	Image cmd_utils.ImageFlags `mapstructure:",squash"`
+	MigrationImage cmd_utils.MigrationImageFlags `mapstructure:",squash"`
```

Comment:
With the parameters change suggested below you only need this:
```suggestion
	MigrationImage cmd_utils.MigrationImageFlags `mapstructure:"migration"`
```